nav_msgs/Path init_path
std_msgs/Int8 generator_mode
std_msgs/Float32 look_ahead
std_msgs/Float32 cruising_speed
---
nav_msgs/Path generated_path